src/gwt/src/org/rstudio/studio/client/rsconnect/ui/RSConnectDeploy.ui.xml

```diff
@@ -33,10 +38,11 @@
          </g:VerticalPanel>
          <g:VerticalPanel ui:field="descriptionPanel_"
                           visible="false">
-            <g:Label text="Publish: "></g:Label>
+            <g:Label><ui:msg key="publishText">Publish: </ui:msg></g:Label>
```

Comment:
To localize attributes (text= in this case) we should use the following syntax. There can be code executed when things are set via attributes and we don't want to skip that.

```suggestion
            <g:Label text="Publish: "><ui:attribute ui:name="text" key="publishText"/></g:Label>
```